tgui/packages/tgui/constants.js

```diff
@@ -54,6 +54,11 @@ export const RADIO_CHANNELS = [
     freq: 1213,
     color: '#a52a2a',
   },
+  {
+    name: 'SyndicateTeam',
+    freq: 1244,
+    color: '#a52a2a',
+  },
```

Comment:
`Syndicate Team` maybe?

Reply:
This was changed to SyndTeam. It has to match exactly the official name of the channel in order for colorizing to work.

Reply:
Fair enough